knative-operator/pkg/controller/knativekafka/knativekafka_controller_test.go

```diff
@@ -436,6 +436,9 @@ func makeCr(mods ...func(*v1alpha1.KnativeKafka)) *v1alpha1.KnativeKafka {
 				Enabled:          false,
 				BootstrapServers: "foo.bar.com",
 			},
+			HighAvailability: &eventingv1alpha1.HighAvailability{
```

Comment:
I got confused by _reading_ `eventingv1alpha1.HighAvailability` - since the HA type is actually on the common spec.

I guess a nit. but perhaps changing the import from:
```
	eventingv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
```
to
```
	operatorv1alpha1 "knative.dev/operator/pkg/apis/operator/v1alpha1"
```
? 

I know it was that way already :see_no_evil:

Reply:
sure thing